cmd/pipeline/deploy.go

```diff
@@ -281,12 +293,18 @@ func waitForResourcesToBeRunning(ctx context.Context, name, namespace string, ti
 					areAllRunning = false
 				}
 				if status == "error" {
-					errors[name] = 1
+					errorsMap[name] = 1
 				}
 			}
-			if len(errors) > 0 {
-				previewEnvURL := getNamespaceURL(ctx, name)
-				return fmt.Errorf("pipeline '%s' deployed with errors. You can check %s for more information", name, previewEnvURL)
+			if len(errorsMap) > 0 {
+				pipelineURL, err := getPipelineURL(ctx, name, namespace)
+				if err != nil {
+					return fmt.Errorf("pipeline '%s' deployed with errors", name)
+				}
+				return errors.UserError{
+					E:    fmt.Errorf("pipeline '%s' deployed with errors", name),
+					Hint: fmt.Sprintf("Follow your pipeline status via the URL:\n    %s", pipelineURL),
```

Comment:
@rberrelleza this is a deploy that failed, which sentence do you propose?

Reply:
I think for the hint I would just put the URL?